docs/buildah-version.md

```diff
@@ -0,0 +1,27 @@
+## buildah-version "1" "June 2017" "buildah"
+
+## NAME
+buildah version - Display the Buildah Version Information.
+
+## SYNOPSIS
+**buildah version**
+[**--help**|**-h**]
+
+## DESCRIPTION
+Shows the following information for the Client: Version, Go Version, Image Spec, Runtime Spec, Git Commit, Build Time, OS, and Architecture.
```

Comment:
We're not a client-server program, so referring to it as a client in the description is confusing.